src/routes/invite/+page.svelte

```diff
@@ -40,28 +41,29 @@
 </script>
 
 <svelte:head>
-    <title>Accept invite - Appwrite</title>
+    <title>{$_.t('accept_invite.title')} - Appwrite</title>
```

Comment:
I forgot, is this typesafe? E.g. if I pass in a string that doesn't exist, will the IDE (not the server) return an error?

Reply:
@TGlide I think now we are working with pull request #459, so we have stopped working on this PR we can close this PR, but first we can have a discussion with Torsten about it first.